docs/designers-developers/developers/tutorials/create-block/submitting-to-block-directory.md

```diff
@@ -0,0 +1,89 @@
+
+**Not So Good**: WebTeam5 Image Works
+**Good**: Responsive Image Slider by WebTeam5
+
+**Question: What happens when there are multiple blocks with similar names?**
+Try your best to make your block's name functional and unique to make it stand out. Look for applicable synonyms or include a prefix if necessary.
+
+### Clearly describe your block
+
+The description really helps to communicate what your block does.The quicker a user understands how your block will help them, the more likely it is a user will use your block. Users will be reading your block's description within the Block Editor where space can be limited. Try to keep it short and concise.
+
+**Not So Good**: The best way to show images on your website using jQuery and CSS.
+**Good**: An easy to use, responsive, Image gallery block. 
+
+**Tip**: It’s not about marketing your block, in fact we want to avoid marketing in blocks. You can read more about it in the [plugin guidelines]. Stick to being as clear as you can. The block directory will provide metrics to let users know how awesome your block is!
+
+### Add Keywords for broader context
+
+Keywords add extra context to your block and make it more likely to be found in the inserter. 
+
+Examples for an Image Slider block:
+- slider
+- carousel
+- gallery
+
+[Read more](https://github.com/WordPress/gutenberg/blob/master/docs/rfc/block-registration.md#keyword) about keywords.
+
+### Choose the right category
+
+The Block Editor allows you to indicate the category your block belongs in, making it easier for users to locate your block in the menu.
+
+**Possible Values**:
+- text
+- media
+- design
+- widgets
+- embed
+
+[Read more](https://github.com/WordPress/gutenberg/blob/master/docs/rfc/block-registration.md#category) about categories.
+
+Wondering where to input all this information? Read the next section :)
+
+## Step 2: Analyze your plugin
+
+The `block.json` file lives in the root folder of your block and provides the Block Directory important information about your block. Along with being the place to store contextual information about your block like the: `name`, `description`, `keywords` and `category`, the `block.json` file stores the location of your block’s files.
+
+Double check that the following is true for your block:
+`editorScript` is pointing to the JavaScript bundle that includes all the code used in the **editor**.
+`editorStyle` is pointing to the CSS bundle that includes all the css used in the **editor**.
+`script` is pointing to the JavaScript bundle that includes all the code used on the **website**.
+`style` is pointing to the CSS bundle that includes all the code used on the **website**.
+Includes example data (Optional, but useful)
+
+Although it isn’t necessary that you have both an `editorScript/editorStyle` and `script/style` listed in your `block.json` we encourage the separation of code used for editing and code used for displaying your block in order to keep both interfaces running quickly.
```

Comment:
I think you could be clearer in stating that at least one .js file is definitely required.

Reply:
What about something like:

"Although blocks require at least one javascript file listed in the `block.json`, we encourage the separation of code used for editing and code used for displaying your block in order to keep both interfaces running quickly.

Reply:
We encourage the separation of code used in the backend and the frontend to keep both interfaces running smoothly by including both the following files: `editorScript/editorStyle` and `script/style` listed in your `block.json`